### PR TITLE
[#IOCOM-1202] Splitted openapi specs in internal and external

### PR DESCRIPTION
**Pull request by michaeldisaro**
#### List of Changes
Splitted openapi specs

#### Motivation and Context
We need to keep some api internal and expose others.

#### How Has This Been Tested?
build and unit tests

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,9 @@
     "test": "jest",
     "test:coverage": "jest --coverage",
     "lint": "eslint . -c .eslintrc.js --ext .ts,.tsx",
-    "openapi:bundle": "bundle-api-spec -i openapi/index.yaml.template -o openapi/index.yaml -V $npm_package_version",
+    "openapi:bundle": "npm-run-all openapi:bundle:*",
+    "openapi:bundle:internal": "bundle-api-spec -i openapi/index_internal.yaml.template -o openapi/index.yaml -V $npm_package_version",
```
Comment (Garma00) [inline package.json:17]: Do we want to keep `index.yaml` as the name of the output file? Or maybe we can name it `index_internal.yaml` just like the other one

File: openapi/index_external.yaml [hunk 2]
```diff
@@ -104,14 +103,48 @@ paths:
         - application/json
     get:
       operationId: listRCConfiguration
-      summary: Get all the remote-content configurations
+      summary: Get all the remote-content configurations associated to the userId
```
Comment (Garma00) [inline openapi/index_external.yaml:106]: Does the external caller have context about the userId?

File: openapi/index_external.yaml [hunk 3]
```diff
@@ -345,10 +384,31 @@ definitions:
         type: string
         minLength: 1
         description: authentication config type
+      cert:
+        $ref: '#/definitions/RCClientCert'
```
Comment (acuiuli) [inline openapi/index_external.yaml:388]: Shouldn't the cert data be removed from the external api?

File: package.json (hunk 1 above)
Comment (michaeldisaro) [reply package.json:17]: I did this to avoid breaking any api that could be pointing it.

File: openapi/index_external.yaml [hunk 4]
```diff
@@ -104,14 +103,46 @@ paths:
...
-        Get all the remote-content configurations
+      summary: Get all the owned remote-content configurations
+      description: Get all the owned remote-content configurations
       responses:
         '200':
           description: Return the configurations
           schema:
             $ref: '#/definitions/RCConfigurationListResponse'
+          examples:
+            text/json: |
```
Comment (lucacavallaro) [inline openapi/index_external.yaml:114]:
```suggestion
            application/json: |
```

**Review by lucacavallaro — CHANGES_REQUESTED** [review]

**Review by lucacavallaro — APPROVED** [review]

**Review by acuiuli — APPROVED** [review]

**Review by Garma00 — APPROVED** [review]

**Pull request merged**